python/tank/platform/__init__.py

```diff
@@ -10,7 +10,7 @@
 
 
 # Engine management
-from .engine import start_engine, current_engine, get_engine_path, find_app_settings
+from .engine import start_engine, current_engine, get_engine_path, find_app_settings, _restart_engine
```

Comment:
why is this import private?

Heads up that in the 0.18 branch, I moved all of the inline methods inside __init__ into a platform.util class as part of the cleanup.